airflow/migrations/versions/d38e04c12aa2_add_serialized_dag_table.py

```diff
@@ -61,6 +61,7 @@ def upgrade():
             table_name="serialized_dag",
             column_name="last_updated",
             type_=mysql.TIMESTAMP(fsp=6),
+            nullable=False,
```

Comment:
Random comment: WHy don't we use DATETIME columns? https://dev.mysql.com/doc/refman/8.0/en/datetime.html

Reply:
It didn't seem to keep time in UTC. So kept it in-line with other tables 

https://github.com/apache/airflow/blob/a7a39003f97451347836acbc205d4ee2c35cbfd6/airflow/migrations/versions/0e2a74e0fc9f_add_time_zone_awareness.py#L54-L57

Reply:
Probably best to keep them all the same for now, but given our SQLA layer I don't think we _can_ write anything other than UTC in to the DB. 

But also I don't really care _what_ we do on Mysql so my caring is low.